external-crates/move/crates/move-compiler/src/typing/core.rs

```diff
@@ -923,7 +923,12 @@ impl<'env> Context<'env> {
             if let Some(names) = scope.use_funs.get(tn) {
                 let mut new_names = names
                     .iter()
-                    .map(|(_, _, use_fun)| use_fun.target_function)
+                    .flat_map(|(_, method_name, use_fun)| {
+                        vec![
+                            (use_fun.target_function, *method_name),
+                            (use_fun.target_function, use_fun.target_function.1.value()),
```

Comment:
Is it possible that this will lead to a problem, such as if this name is shadowed by a different `use fun`?